stow/style-editor.css

```diff
@@ -1182,6 +1182,17 @@ pre.wp-block-verse {
 	color: var(--wp--preset--color--background);
 }
 
+.has-tertiary-background-color,
+.has-tertiary-background-color.has-background-dim {
+	background-color: var(--wp--preset--color--tertiary);
+	color: var(--wp--preset--color--background);
+}
+
+.has-tertiary-background-color:not(.has-text-color),
+.has-tertiary-background-color.has-background-dim:not(.has-text-color) {
+	color: var(--wp--preset--color--foreground);
+}
+
```

Comment:
Stow is a Varia child theme, we make the css changes in the sass files and build using `npm run start` or `npm run build`

Reply:
in this case I think this should be changed directly in varia, since the child theme is importing the partial from the parent, but it's weird that we need this only for Stow then, if none of the other Varia themes are using it.

Reply:
To help you test, you can use the markup on this post: pcrugM-jc-p2l give you access to that demo site

Reply:
Varia doesn't have a tertiary definition, so the missing styles in child themes.
What shall be the spec for the colors if we need to introduce `tertiary`. I couldn't find in figma.

<img width="372" alt="image" src="https://user-images.githubusercontent.com/1935113/158152498-542051ef-a4db-4cd9-938d-0cb253b6a5d0.png">

@kjellr can you help here.

Reply:
this would be the tertiary color for stow: https://github.com/Automattic/themes/blob/15bb10aeb42cd3aa477ce755fe35048f7cdbbfce/stow/functions.php#L19 I think we are using different naming conventions on Varia where `tertiary` == `border` https://github.com/Automattic/themes/blob/15bb10aeb42cd3aa477ce755fe35048f7cdbbfce/stow/sass/_global-variables.scss#L14

Reply:
We probably need to add the new CSS rules here: https://github.com/Automattic/themes/blob/15bb10aeb42cd3aa477ce755fe35048f7cdbbfce/varia/sass/blocks/utilities/_style.scss#L117 and make sure that the name of the class is the same as the one the editor is outputting. It's very strange that this is happening now (Varia themes have been around for a few years and this feels like a big oversight). I'm thinking this bug might have been created by a refactor we did to move from sass variables to CSS variables for some of the varia children and something got lost in the process. This is the PR that did that for Stow: https://github.com/Automattic/themes/pull/3241

Reply:
Be sure to test any changes here with the Customizer's ability to change colors. The color annotations in Varia + its children are very complicated and prone to breaking.

Reply:
Updated Varia theme here: https://github.com/Automattic/themes/pull/5666